src/assets/i18n/en.json

```diff
@@ -1654,6 +1654,8 @@
     "car_makers": "Car Makers",
     "create_success": "The car '{{carName}}' has been created with success",
     "update_success": "Car updated successfully",
+    "delete_success": "Car deleted successfully",
```

Comment:
Follow what exists when delete/update is successful:
    "update_success": "The user '{{userFullName}}' has been updated successfully",
    "delete_success": "The user '{{userFullName}}' has been deleted successfully",
As usual, don't reinvent the wheel.